examples/src/main/r/0-getting-started.R

```diff
@@ -0,0 +1,23 @@
+#
+# Author:   Daniel Emaasit (@emaasit)
+# Purpose: This script shows how to install SparkR onto your workstation/PC
+#          and initialize a spark context and a SparkSQL context
+# Date:    06/05/2015
+#
+
+
+# Install SparkR from CRAN
+install.packages("SparkR")
```

Comment:
In the case of these example files we will be assuming the user has installed Spark through some method (say from http://spark.apache.org/downloads.html). So we can just include the package as `library(SparkR)` and avoid the installation steps.